app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Resend confirmation instructions" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Great job so far 👏🏻  however The views in devise for the registration and login should be modified to match the wireframes and their styling. Kindly address this 👇🏻 


![image](https://user-images.githubusercontent.com/79058364/200050396-faeb7f4d-c14a-40ee-9883-a35af9a69e38.png)

- Your Design:

![Screenshot from 2022-11-04 20-28-58](https://user-images.githubusercontent.com/79058364/200051096-12dba9ac-2815-47a9-9e2b-89db0dab7f3b.png)
![Screenshot from 2022-11-04 20-29-12](https://user-images.githubusercontent.com/79058364/200051112-055c6c16-3764-4cfa-94f6-1996a30a0e47.png)